pkg/sfu/downtrack.go

```diff
@@ -859,7 +942,7 @@ func (d *DownTrack) writeSimpleRTP(extPkt *buffer.ExtPacket) error {
 	}
 	if d.sequencer != nil {
 		meta := d.sequencer.push(extPkt.Packet.SequenceNumber, newSN, newTS, 0, extPkt.Head)
-		if meta != nil && d.vp8Munger != nil {
+		if meta != nil && translatedVP8 != nil {
```

Comment:
The translated header can be `nil` for padding only packets. So, checking for that.